go/libraries/doltcore/sqle/enginetest/dolt_queries_schema_merge.go

```diff
@@ -551,6 +563,62 @@ var SchemaChangeTestsBasicCases = []MergeScriptTest{
 			},
 		},
 	},
+	{
+		Name: "ambiguous choice of ancestor column",
+		AncSetUpScript: []string{
+			"CREATE table t (pk int primary key, col1 int, col2 int);",
+			"INSERT into t values (1, 10, 100), (2, 20, 200);",
+		},
+		RightSetUpScript: []string{
+			"alter table t drop column col2;",
+			"insert into t values (3, 30), (4, 40);",
+		},
+		LeftSetUpScript: []string{
+			"alter table t drop column col1;",
+			"alter table t rename column col2 to col1;",
+			"insert into t values (5, 50), (6, 60);",
+		},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Skip:           true,
```

Comment:
It's skipped because it triggers https://github.com/dolthub/dolt/issues/7767. I added a comment explaining why its skipped.

This will probably be fixed if we get rid of column tags.